packages/js/ai/src/utils/requestJetpackToken.ts

```diff
@@ -31,57 +31,12 @@ declare global {
  */
 
 export async function requestJetpackToken() {
```

Comment:
Should we mark this as deprecated? It'd also be nice to notify other teams somehow to replace it with the Jetpack package.

Reply:
I've added a @deprecated JSDoc tag and a console.warn() when the function is called, I'll see how we could go about notifying folks. 👍